pages/common/arc.md

```diff
@@ -0,0 +1,20 @@
+# arc
+
+> Arcanist(Arc) is a command-line interface to Phabricator.
+> More information <https://secure.phabricator.com/book/phabricator/article/arcanist/>.
+
+- Send your code to Differential for review:
```

Comment:
What code is sent? Generally we prefer to use the 3rd person and avoid `your`, `I`, etc. in tldr-pages.

Reply:
Sorry, it is the word used on the arc site. How about `Send changes to Differential for review` ?

Reply:
Yeah, that would work @einverne! Thanks.

Sometimes the project's actual wording takes a bit of tweaking to get it to look good in a tldr page :-)